Taskfile.yml

```diff
@@ -228,11 +228,12 @@ tasks:
   fixtures:load:
     prompt: "This will reset your content. Continue?"
     cmds:
-     - task drush -- --yes pm:enable hoeringsportal_base_fixtures $(find web/modules/custom -type f -name 'hoeringsportal_*_fixtures.info.yml' -exec basename -s .info.yml {} \;)
+     # - task drush -- --yes pm:enable hoeringsportal_base_fixtures $(find web/modules/custom -type f -name 'hoeringsportal_*_fixtures.info.yml' -exec basename -s .info.yml {} \;)
```

Comment:
remove?

Reply:
Fixed in https://github.com/itk-dev/hoeringsportal/pull/462/commits/f2a05220af836cf4328320e878d316d6ce88c1cb.